src/components/studies/StudyList.tsx

```diff
@@ -184,6 +204,10 @@ const StudySublist: FunctionComponent<StudySublistProps> = ({
               }}
               isNewlyAddedStudy={highlightedStudyId === study.identifier}
               section={item.sectionStatus}
+              isMenuOpen={
+                menuAnchor !== null &&
+                menuAnchor.study.identifier === study.identifier
```

Comment:
I think menuAnchor?.study?.identifier === study.identifier is sufficient because if it will resolve to undefined it will not be equal (dont need menuAnchor !== null)